geo_concerns/app/models/concerns/geo_concerns/image_behavior.rb

```diff
@@ -0,0 +1,31 @@
+module GeoConcerns
+  module ImageBehavior
+    extend ActiveSupport::Concern
+    include ::CurationConcerns::GenericWorkBehavior
+    include ::CurationConcerns::BasicMetadata
+    include ::GeoConcerns::BasicGeoMetadata
+
+    included do
+      aggregates :rasters, predicate: RDF::Vocab::ORE.aggregates,
+                           class_name: 'GeoConcerns::Vector',
+                           type_validator: type_validator
+      filters_association :members, as: :image_files, condition: :concerns_image_file?
```

Comment:
> I just download a fresh copy of your repo, started a new gemset, and now the tests all pass.

Understood, thank you.

> However, I'm not quite sure why.

This is warranted.  It's definitely not an issue with the FactoryGirl implementation:
```
    factory :image_with_rasters do
      before(:create) do |image, evaluator|
        2.times { image.rasters << FactoryGirl.create(:raster, user: evaluator.user) }
      end
    end
```
...as this *is* appending the Raster Objects.

Hence, ActiveFedora::Associations should be raising an error within the scope of the above referenced factory implementation.

I'll inspect the code for other such oversights.  Thank you for bringing this to my attention.